client/app/views/TrackViewer/Sidebar.vue

```diff
@@ -143,7 +154,15 @@ export default defineComponent({
           @track-split="$emit('track-split', $event)"
           @track-seek="$emit('track-seek', $event)"
           @update-new-track-settings="$emit('update-new-track-settings',$event)"
-        />
+        >
+          <template>
```

Comment:
Settings tab wouldn't open and closed, it was a named slot, this should fix.
```suggestion
          <template slot="settings">
```